terraform/modules/hub/modules/ecs_asg/files/cloud-init.sh

```diff
@@ -190,6 +216,15 @@ docker run \
   --env="ECS_LOGLEVEL=warn" \
   ${ecs_agent_image_identifier}
 
+ExecStop=/usr/bin/docker stop -t 120 ecs-agent
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+systemctl daemon-reload
+systemctl enable ecs
```

Comment:
```suggestion
systemctl enable ecs
systemctl start ecs
```

Can we start ecs service immediately?

Reply:
See the commit message and the PR description. The whole point of this change is to not start the ECS container during clout-init (i.e. until after the reboot that happens at the bottom of the script).